pkgs/by-name/co/copilot-language-server/package.nix

```
@@ -24,30 +25,47 @@ let
     }
     ."${stdenvNoCC.hostPlatform.system}"
       or (throw "Unsupported system: ${stdenvNoCC.hostPlatform.system}");
-in
 
+  executableName = "copilot-language-server";
+  fhs =
+    { package }:
+    buildFHSEnv {
+      name = package.meta.mainProgram;
+      version = package.version;
+      targetPkgs = pkgs: [ pkgs.stdenv.cc.cc.lib ];
+      runScript = "${package}/bin/${package.meta.mainProgram}";
```

```suggestion
      runScript = lib.getExe package;
```

I think this should work too.

---

Totally right, I tried to reinvent the wheel there 😂